spec/features/editing_file_attachments/edit_file_attachment_file_spec.rb

```diff
@@ -0,0 +1,35 @@
+RSpec.feature "Edit a file attachment file", js: true do
+  scenario do
+    given_there_is_an_edition_with_featured_attachments
+    when_i_go_to_edit_an_attachment
+    and_i_add_a_unique_reference
+    then_i_see_the_timeline_entry
```

Comment:
Can we add a step here to check we can see the new metadata? We should [now have the component ready](https://trello.com/c/UET4vx6k/1439-update-attachment-component-to-accommodate-new-attachment-design) to do this.

Reply:
Also spotted the name of this file is inaccurate - we should change it to something like `edit_file_attachment_spec`.

Reply:
Updated.